### DISCUSSION TITLE
Thermostats - HmIP-BWTH - always jumps from auto mode to manuel mode

### DISCUSSION THREAD

**Discussion by tux1980** [OpenCCU/OpenCCU#2701] [🇺🇸 Q&A]
### Describe the issue you are experiencing

Hi,

quite a time now I'm experience an issue with my HmIP-BWTH Devices (especially with two out if seven). I'm using RaspberryMatic in Version 3.75.6.20240316 on a Raspberry 4+.
The thermostat is sets to "Auto Mode" since I'm using  schedulers, changing the temperature 3 times a day (Screenshot attached). 
![Screenshot from 2024-03-27 07-30-47](https://github.com/jens-maus/RaspberryMatic/assets/46625474/78b9f159-b4d9-4fae-b85f-5a513b98250f)
![Screenshot from 2024-03-27 07-31-03](https://github.com/jens-maus/RaspberryMatic/assets/46625474/9597755f-e523-4809-b3ac-de15f23a9b36)

Problem is, after a while, the thermostates are reverting themselves back  from "auto mode" to "manu mode" and put themsevles to the minimal temperature to 5 degrees. 
![Screenshot from 2024-03-27 07-33-47](https://github.com/jens-maus/RaspberryMatic/assets/46625474/7911cba0-f37b-4a49-b5b7-256883c01185)

Thanks for helping us out
Wolfgang

### Describe the behavior you expected

* That it keeps the ssettings to "auto mode" and not automatically reverts to "manu mode" again.

### Steps to reproduce the issue

1. Setting back from "manu" to "auto mode" again - after a while it reverts its settings again to manu mode.
2.
3.
...

### What is the version this bug report is based on?

3.75.6.20240316

### Which base platform are you running?

rpi4 (RaspberryPi4, ARM64/aarch64)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
What File is useful here?
```

**Comment by jens-maus** [comment]
As you a rather seeking for help and not reporting a bug/issue in RaspberryMatic, this does not belong here but in the discussion area. Thus, will move it over so that someone could potentially see if there might be a solution for your local issue.
